pkg/controller/dnsendpoint/nameserver/gcp_live_test.go

```diff
@@ -139,3 +145,26 @@ func (s *LiveGCPTestSuite) getCUT() *gcpQuery {
 		},
 	}
 }
+
+func (s *LiveGCPTestSuite) testCreateThenUpdate(tc *testCreateAndDeleteCase) {
+	cut := s.getCUT()
+	domain := fmt.Sprintf("live-gcp-test-%08d.%s", rand.Intn(100000000), s.rootDomain)
+	s.T().Logf("domain = %q", domain)
+	err := cut.Create(s.rootDomain, domain, sets.NewString(tc.createValues...))
+	if s.NoError(err, "unexpected error creating NS") {
+		defer func() {
+			err := cut.Delete(s.rootDomain, domain, sets.NewString(tc.deleteValues...))
+			s.NoError(err, "unexpected error deleting NS")
+		}()
+	}
+
+	// now test updating by re-issuing a Create()
+	err = cut.Create(s.rootDomain, domain, sets.NewString(tc.createValues...))
```

Comment:
Change this to use a `tc.updateValues`, and add a case where the update is changing the values.